agnostic-svelte-ts/.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	useTabs: true,
+	singleQuote: true,
+	trailingComma: 'none',
```

Comment:
Non-blocking: I actually prefer trailing commas but not a blocker haha

Reply:
Yeah I also couldn't really decide here, I personally prefer trailing commas too. 😄 On one hand the SvelteKit installer generated the 'none' rule, on the other I looked at the other projects, and all of them used trailing commas, so I went with the SvelteKit default. I think it's a good question in general whether the subprojects should have the exact same formatting ruleset or not?

Reply:
Good point. I think they may have different idiomatic conventions